packages/core/lib/commands/console/meta.js

```diff
@@ -18,7 +18,7 @@ module.exports = {
       {
         option: "--url",
         description:
-          "Creates a provider and connect to the network. This can be used outside of a Truffle project."
+          "Creates a provider and connects to the network. This can be used outside of a Truffle project."
```

Comment:
maybe we should specify "using the given url"? do you think that makes for a clearer, more complete description of this option?

Reply:
I agree @eggplantzzz . Specifying this will make it more clearer. I will change the description.